test/unit/lib/getPharmacies.js

```diff
@@ -47,14 +48,48 @@ describe('Nearby', () => {
       });
     });
 
-    it('should get the number of results requested, ordered by distance', () => {
-      const requestedNumberOfResults = 5;
-      const results = pharmacies.nearby(searchPoint, geo, requestedNumberOfResults).nearbyServices;
+    it('should get the number of unique nearby open services requested, ordered by distance',
```

Comment:
This seems to be testing 3 distinct things (number, uniqueness and distance).

It might be better to use an enclosing describe ('get open services') with 3 it() clauses.

There is probably also a need for a 4th it clause to expect on openness.